ui/root/src/commonMain/kotlin/social/androiddev/root/root/RootComponent.kt

```diff
@@ -49,4 +48,10 @@ interface RootComponent {
     sealed interface DeepLink {
         object None : DeepLink
     }
+
+    sealed class UiAuthStatus {
```

Comment:
Please move this to the same file as the composable or to its own file. It doesn't need to be coupled with the component file namespace

Reply:
Also is unknown ever used?

Reply:
Sure thing. Yes, `unknown` is used as initial value

Reply:
Ah right, I missed that. Might be clearer to rename to UIAuthStatus.Loading or make nullable instead.

Reply:
I like `Loading` better 👍